languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/producers/JavaModelClassFileProducer.kt

```diff
@@ -148,7 +148,7 @@ class JavaModelClassFileProducer @Inject constructor(override val vrapTypeProvid
                     }else if(it.name.equals("interface")) {
                         "@JsonProperty(\"${it.name.lowerCamelCase()}\") final ${it.type.toVrapType().fullClassName()} _interface"
                     } else {
-                        "@JsonProperty(\"${it.name.lowerCamelCase()}\") final ${it.type.toVrapType().fullClassName()} ${it.name.lowerCamelCase()}"
+                        "@JsonProperty(\"${it.name}\") final ${it.type.toVrapType().fullClassName()} ${it.name.lowerCamelCase()}"
```

Comment:
That looks interesting... Did this ever work and how did you find this problem?

Reply:
I found it because I couldn't deserialize Message. I have no idea how it worked before.